spring-session/src/main/java/org/springframework/session/data/couchbase/DelegatingSessionStrategy.java

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I don't think this should be needed. I think this is probably due to the global support which I think we should remove. If we do add global session support, we would move it to a different PR.